src/multicall/index.ts

```diff
@@ -0,0 +1,41 @@
+import multicallEvm from './evm';
+import multicallStarknet from './starknet';
+import networks from '../networks.json';
+
+const STARKNET_CHAIN_IDS = ['0x534e5f4d41494e', '0x534e5f5345504f4c4941'];
```

Comment:
would be nice to get this from networks.json (maybe a new param ?) so everything will be in one place